src/pages/EditMonitor.vue

```diff
@@ -422,6 +422,62 @@ export default {
 };
 </script>
 
+<style src="vue-multiselect/dist/vue-multiselect.css"></style>
```

Comment:
This part, if I am correct, it should be removed in #511 .

Reply:
Line 425 - 479

Reply:
yes you're right, i just noticed.
I'll fix it

Reply:
fixed